flink-table/flink-table-api-java-bridge/src/main/java/org/apache/flink/table/sources/CsvBatchTableSourceFactory.java

```diff
@@ -19,18 +19,18 @@
 package org.apache.flink.table.sources;
 
 import org.apache.flink.annotation.PublicEvolving;
-import org.apache.flink.table.factories.BatchTableSourceFactory;
+import org.apache.flink.table.factories.StreamTableSourceFactory;
 import org.apache.flink.types.Row;
 
 import java.util.Map;
 
 /** Factory for creating configured instances of {@link CsvTableSource} in a batch environment. */
 @PublicEvolving
 public class CsvBatchTableSourceFactory extends CsvTableSourceFactoryBase
```

Comment:
What about the name of this class? This seems like a misnomer now?

Reply:
I think this class will also be dropped soon. I kept the term "batch" here because it is intended for batch mode.